install_config/topics/configuring_for_aws_registry-proc.adoc

```diff
@@ -80,12 +80,18 @@ openshift_hosted_registry_pullthrough=true
 openshift_hosted_registry_acceptschema2=true
 openshift_hosted_registry_enforcequota=true
 openshift_hosted_registry_replicas=3
+# AWS Registry Encryption Configuration
+openshift_hosted_registry_storage_s3_encrypt
+openshift_hosted_registry_storage_s3_kmskeyid
```

Comment:
I'm not sure if you can have variables without any value in Ansible inventory files.

@mtnbikenc you made changes in our playbooks recently, is it valid syntax?

Reply:
@mtnbikenc @dmage I dug a little and added what I could about this key id (including annotation), but not sure if there is an actual value that should be included here instead?

Reply:
:+1:, looks good